x-pack/plugin/search-business-rules/src/main/java/org/elasticsearch/xpack/searchbusinessrules/PinnedQueryBuilder.java

```diff
@@ -49,56 +58,184 @@
     public static final int MAX_NUM_PINNED_HITS = 100;
 
     private static final ParseField IDS_FIELD = new ParseField("ids");
+    private static final ParseField DOCUMENTS_FIELD = new ParseField("documents");
```

Comment:
I think it'd be nice to use "docs" instead of "documents" to match our multi-get and multi-term vectors requests.